.eslintrc.js

```diff
@@ -82,6 +82,10 @@ const config = {
         argsIgnorePattern: "^_",
       },
     ],
+    "import/no-extraneous-dependencies": [
+      "off",
+      { devDependencies: ["**/*.test.js", "**/*.spec.js"] },
```

Comment:
Should be *.ts[x] as well no? Or maybe better (taken from #695):

```json
"import/no-extraneous-dependencies": [
    "error",
    { devDependencies: ["**/*.test.*"] },
],
```

Reply:
Fixed